r/adbcsnowflake/DESCRIPTION

```diff
@@ -17,9 +17,9 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
```

Comment:
Is this not missing a lower bound when you mention below that this will not compile on <4.2?
```suggestion
Depends: R (>= 4.2)
Roxygen: list(markdown = TRUE)
```